src/test/java/lotto/LottosTest.java

```diff
@@ -0,0 +1,22 @@
+package lotto;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+public class LottosTest {
+
+    @ParameterizedTest
+    @CsvSource(value = {"14000:14", "13500:13"}, delimiter = ':')
+    void 발급가능수량계산(String input, String expected) {
+        List<Lotto> result = new ArrayList<>();
+        for (int i = 0; i < Integer.parseInt(input) / Lottos.LOTTO_AMOUNT; i++) {
+            result.add(Lotto.purchase());
+        }
+        assertThat(new Lottos(result).count()).isEqualTo(Integer.parseInt(expected));
```

Comment:
로직 구현으로 검증하기 보다는 메소드 테스트로 진행해보아요.
```suggestion
        Lottos actual = Lottos.of(input);
        
        assertThat(actual.발급몇개됐니?()).isEqualTo(expected);
```